.travis.yml

```diff
@@ -6,7 +6,7 @@ jdk:
 
 sudo: false
 
-script: ./gradlew test -s
+script: ./gradlew test -s --info
```

Comment:
gradle output was quite poor, it seemed to be quite helpful for me. But it is fine if you want remove it.